src/pages/patientView/PatientViewPage.tsx

```diff
@@ -115,22 +112,34 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
         //TODO: this should be done by a module so that it can be reused on other pages
         const qs = queryString.parse((window as any).location.search);
 
+        const reaction1 = reaction(
```

Comment:
this may be temporary.  the url route is now modelled in it's own store.  page store could be made to observe this directly, but for now I decided just to wire them up this way.